src/repo_config.rs

```diff
@@ -328,46 +328,46 @@ impl<'a> RepoConfig<'a> {
                         Some(method) => method,
                         None => return Err(Error::MissingMethod(pattern.clone())),
                     },
-                    LookupResult::WrongType => return Err(Error::InvalidMethod(pattern.clone())),
-                    LookupResult::Value(v) => match v {
+                    LookupResult::StringValue(v) => match v {
                         "ansible" => DeployMethod::Ansible,
                         "makefile" | "make" => DeployMethod::Makefile,
                         _ => return Err(Error::InvalidMethod(pattern.clone())),
                     },
+                    _ => return Err(Error::InvalidMethod(pattern.clone())),
                 };
 
                 let playbook = match lookup_as_string(config, "playbook") {
                     LookupResult::Missing => default_playbook.clone(),
-                    LookupResult::WrongType => return Err(Error::InvalidPlaybook(pattern.clone())),
-                    LookupResult::Value(v) =>
+                    LookupResult::StringValue(v) =>
                         match VerifiedPath::file(Some(project_root), Path::new(v)) {
                             Ok(v) => Some(v),
                             Err(_) => return Err(Error::InvalidPlaybook(pattern.clone())),
                         },
+                    _ => return Err(Error::InvalidPlaybook(pattern.clone())),
                 };
                 let inventory = match lookup_as_string(config, "inventory") {
                     LookupResult::Missing => default_inventory.clone(),
-                    LookupResult::WrongType => return Err(Error::InvalidInventory(pattern.clone())),
-                    LookupResult::Value(v) =>
+                    LookupResult::StringValue(v) =>
                         match VerifiedPath::file(Some(project_root), Path::new(v)) {
                             Ok(v) => Some(v),
                             Err(_) => return Err(Error::InvalidInventory(pattern.clone())),
                         },
+                    _ => return Err(Error::InvalidInventory(pattern.clone()))
                 };
 
-                let notify_url = match lookup_as_string(config, "notify_url") {
+                let notify_url = match lookup_as_array(config, "notify_url") {
```

Comment:
Also should change the config entry to look for a `notifiers` or  `notify_urls` key.